### DISCUSSION TITLE
S3 bucket naming should have some randomness

### DISCUSSION THREAD

**Discussion by HajoV** [artilleryio/artillery#2131] [Artillery]
Hi there,

While running on AWS (fargate or Lambda), the naming of the S3 bucket is always exaclty the same... so if you happen to delete it (my mom taught me to clean up behind my b#d) and you're willing to start a new test, you will have to wait some time until the deletion is known in the entire AWS organization.

Creating a S3 bucket with the same name as one you deleted moments before leads to the following error : "A conflicting conditional operation is currently in progress against this resource. Please try again".

Version info:
```
2.0.0-36
```

Running this command: 
```
artillery run-fargate --region eu-west-1 foo.yml
```

I expected to see this happen:

"...test being started..."

Instead, this happened:

"    Operation Aborted: A conflicting conditional operation is currently in progress against this resource. Please try again.
    Code: Operation Aborted"

Files being used:
```

**Comment by hassy** [comment]
Hi @HajoV 👋 There's no need to manually delete the S3 bucket between test runs. It's kept around by design.
